apps/nextjs-website/src/components/organisms/Auth/SignUpForm.tsx

```diff
@@ -0,0 +1,354 @@
+                    inputProps={{
+                      sx: {
+                        padding: '8.5px 14px',
+                      },
+                    }}
+                  />
+                  <FormHelperText error={isPasswordDirty && !isPasswordValid}>
+                    {signUp.passwordPolicy}
+                  </FormHelperText>
+                </FormControl>
+              </Stack>
+              <Stack spacing={2} mb={2}>
+                <FormControl variant='outlined'>
+                  <InputLabel
+                    htmlFor='confirm-password-input'
+                    sx={{ top: '-8px' }}
+                  >
+                    {shared.confirmPassword}
+                  </InputLabel>
+                  <OutlinedInput
+                    id='confirm-password-input'
+                    type={showPassword ? 'text' : 'password'}
+                    required
+                    onChange={(e) => setConfirmPassword(e.target.value)}
+                    endAdornment={
+                      <InputAdornment position='end'>
+                        <IconButton
+                          aria-label='toggle confirm password visibility'
+                          onClick={handleClickShowConfirmPassword}
+                          onMouseDown={handleMouseDownConfirmPassword}
+                          edge='end'
+                        >
+                          {showConfirmPassword ? (
+                            <VisibilityOff />
+                          ) : (
+                            <Visibility />
+                          )}
+                        </IconButton>
+                      </InputAdornment>
+                    }
+                    value={confirmPassword}
+                    label={shared.confirmPassword}
+                    error={isPasswordDirty && password !== confirmPassword}
+                    inputProps={{
+                      sx: {
+                        padding: '8.5px 14px',
+                      },
+                    }}
+                  />
+                </FormControl>
+              </Stack>
+              <Stack spacing={2} mb={2}>
+                <FormControl fullWidth>
+                  <InputLabel id='company-field' sx={{ top: '-8px' }}>
+                    {shared.company}
+                  </InputLabel>
+                  <Select
+                    labelId='company-field'
+                    id='company-field-select'
+                    value={company}
+                    label={shared.company}
+                    onChange={(e) => setCompany(e.target.value)}
+                    sx={{ padding: '8.5px 14px' }}
+                    inputProps={{
+                      sx: {
+                        padding: 0,
+                      },
+                    }}
+                  >
+                    <MenuItem value='a'>a</MenuItem>
+                    <MenuItem value='b'>b</MenuItem>
+                    <MenuItem value='c'>c</MenuItem>
```

Comment:
There is yet a task to manage this company select ? :thinking: I think is better adding a TODO here and maybe remove this mock value and disabled field for now